File: airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresRawOverrideTypingDedupingTest.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.destination.postgres.typing_deduping;

import com.fasterxml.jackson.databind.node.ObjectNode;

public class PostgresRawOverrideTypingDedupingTest extends PostgresTypingDedupingTest {

  @Override
  protected ObjectNode getBaseConfig() {
    return super.getBaseConfig()
        .put("raw_data_schema", "overridden_raw_dataset");
  }

  @Override
  protected String getRawSchema() {
    return "overridden_raw_dataset";
  }

}

File: airbyte-integrations/connectors/destination-postgres/src/test-integration/kotlin/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresRawOverrideTypingDedupingTest.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.integrations.destination.postgres.typing_deduping

import com.fasterxml.jackson.databind.node.ObjectNode

class PostgresRawOverrideTypingDedupingTest : PostgresTypingDedupingTest() {
    override fun getBaseConfig(): ObjectNode {
        return super.getBaseConfig().put("raw_data_schema", "overridden_raw_dataset")
    }

    override val rawSchema: String
        get() = "overridden_raw_dataset"
}
